diambra/arena/arena_gym.py

```diff
@@ -45,89 +47,42 @@ def __init__(self, env_settings):
                                             dtype=np.uint8)
 
     # Processing Environment info
-    def env_info_process(self, env_info):
+    def env_info_process(self, env_info_dict):
```

Comment:
Similar q as in engine: Why do we need to translate these and not use the protobuf generated structs directly?

Reply:
let's address this in the next PR